canonical-web-and-design/microk8s.io
# Add and initial run of inclusive naming action

## Done

- Adds inclusive naming github action
- Remove any previously existing non-inclusive language

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8027/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check github action works
- Check language changes make lexical sense


## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4453
